docker/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM alpine:3.5
+LABEL maintainer Jan Koppe <post@jankoppe.de>
+
+ARG PUMPIO__GUID=1337
```

Comment:
Not sure that it makes a difference practically but these should probably be system users (<1000), though I do appreciate the choice of 1337 :)